lib/config.go

```diff
@@ -68,6 +68,7 @@ func flagsToConfig(f *flag.FlagSet) (*Config, error) {
 	f.StringVar(&cfg.CDNDistributionID, "distribution-id", "", "optional CDN distribution ID for cache invalidation")
 	f.StringVar(&cfg.ConfigFile, "config", ".s3deploy.yml", "optional config file")
 	f.IntVar(&cfg.MaxDelete, "max-delete", 256, "maximum number of files to delete per deploy")
+	f.BoolVar(&cfg.PublicReadACL, "s3-public-acl", false, "set public ACL on uploaded objects, defaults to private if not set.")
```

Comment:
I have tried to avoid using "s3" in the other flags. Not that I think that this tool will ever be more general, but I think it looks nicer.

So `public-access`? Or maybe `enable-public-access`?

The rest of this PR looks good.